media/js/firefox/australis/growth-firstrun-test1.js

```diff
@@ -0,0 +1,217 @@
+                    $fxaDelayedCTAs = $('#fxa-delayed-ctas').detach();
+
+                    $fxaOverlay.prepend($fxaValueProp).append($fxaDelayedCTAs);
+
+                    showFxA();
+                }
+                // oauth_complete sent after user verifies account
+                else if (event.data.indexOf('oauth_complete') > -1) {
+                    window.gaTrack(['_trackEvent', '/firstrun/ Optimization f35', 'Sign Up for Firefox Account', 'Verified Email Address']);
+                }
+            } else if (typeof event.data === 'object') {
+                if (event.data.addon) {
+                    window.gaTrack(['_trackEvent', '/firstrun/ Optimization f35', 'Add to Firefox', event.data.addon]);
+                }
+            }
+        });
+
+        // if chosen to show doorhanger, get tour setup
+        if (showDoorhanger) {
+            // Query if the UITour API is working before we start the tour
+            Mozilla.UITour.getConfiguration('sync', function() {
+                // if user has submitted newsletter don't show the tour again
+                if (window.location.hash !== '#footer-email-form') {
+                    // conditionally show doorhanger/tour mask
+                    if (showDoorhanger) {
+                        // id is used for Telemetry
+                        var tour = new Mozilla.BrowserTour({
+                            id: $('#tour-page').data('telemetry'),
+                            startTour: function() {
+                                // prepare FxA stuff
+                                initFxA();
+
+                                // enable AMO iframe
+                                enableIframe();
+
+                                // give FxA a moment to initialize before getting rid of mask
+                                setTimeout(function() {
+                                    tour.doCloseTour();
+                                }, 600);
+                            },
+                            cancelTour: function() {
+                                tour.doCloseTour();
+
+                                showDefaultContent();
+
+                                if ($fxaOverlay) {
+                                    $fxaOverlay.fadeOut('fast');
+                                }
+                            }
+                        });
+
+                        tour.init();
+                    } else {
+                        $body.addClass('noscroll');
+                    }
+                }
+            });
+        // if not showing tour, just get to FxA UI
+        } else {
+            // prepare FxA stuff
+            initFxA();
+
+            // initialize Ghostery iframe
+            enableIframe();
+        }
+
+        // wire up CTAs
+        $('#highlight-sync').on('click', function(e) {
+            e.preventDefault();
+
+            Mozilla.UITour.showHighlight('accountStatus', 'wobble');
```

Comment:
As this may be the first highlight when a user opens the browser, should probably double up this call to work around [Bug 1049130](https://bugzilla.mozilla.org/show_bug.cgi?id=1049130)